### DISCUSSION TITLE
website out out date for 1.19

### DISCUSSION THREAD

**Discussion by Danjhop4** [jacobsjo/snowcapped#17] [Q&A] [answered]
The website is out of date and can you add a way to show how to use it

**Comment by jacobsjo** [comment, marked as answer]
The tool is not out of date. In fact it has just recently been updated to also support custom density functions in 1.19.

For an overview of how to use Snowcapped you can take a look at the wiki: [https://github.com/jacobsjo/snowcapped/wiki](https://github.com/jacobsjo/snowcapped/wiki)

**Reply by Danjhop4** [reply]
I use a custom set of values for the caves

**Reply by Danjhop4** [reply]
1.498 and 1.5 depth for cave biomes

**Reply by jacobsjo** [reply]
So you want to use depth values outside to currently possible range? That would be a vaild feature request that you can open as an issue if you want.

**Comment by Danjhop4** [comment]
Plus it keeps adding mushroom islands

**Reply by jacobsjo** [reply]
Can you please be more specific about this? When does it add mushroom islands?

**Comment by Danjhop4** [comment]
think i worked out why closest value so it picked that as i did not define the biome
